Add data warehouse supported section (#738)
dataform-co/dataform: content/docs/warehouses-supported/snowflake.md

---
title: Snowflake
---

# Authentification

Snowflake connections require the following elements:

- Account name: the first part of your Snowflake url, including region: `account-name.region`.snowflakecomputing.com
- Username and password used for your Snowflake console. You may prefer to create a separate username and password for Dataform to use - please contact our team if you need help.
- Warehouse name: Click "Warehouses" from within your Snowflake console to view a list of warehouses. Any warehouse in your account will work with any database.
- Database: Click "Databases" from within your Snowflake console to view a list of databases.

> You may also need to whitelist Dataform's IP addresses: `35.233.106.210` and `104.196.10.242`

Reviewer: Seems duplicative of below, should we remove this?

<div className="bp3-callout bp3-icon-info-sign bp3-intent-warning" markdown="1">
You may also need to whitelist Dataform's IP addresses to let Dataform access your Snowflake instanced. Please
  follow
  <a
    target="_blank"
    rel="noopener"
    href="https://docs.snowflake.com/en/user-guide/network-policies.html"
  >these instructions</a>. Dataform's IP addresses are <code>35.233.106.210</code> and <code>104.196.10.242</code>.
</div>

# Configuration options

## Querying different databases

> You can both read from and publish to two separate snowflake databases within a single Dataform project. For example, you may have a database called `RAW_SOURCE` that contains raw data loaded in your warehouse and a database called `ANALYTICS` in which you create data tables you use for analytics and reporting.

Reviewer: ```suggestion
You can read from and publish to an unlimited number of snowflake databases within a single Dataform project. For example, you may have a database called `RAW_SOURCE` that contains raw data loaded in your warehouse and a database called `ANALYTICS` in which you create data tables you use for analytics and reporting.
```
